File: app/src/main/java/com/example/dagger_hilt/data/datasource/web/models/MoviesResponse.java
package com.example.dagger_hilt.data.datasource.web.models;

import com.google.gson.annotations.SerializedName;

import java.util.List;

public class MoviesResponse {
    @SerializedName("results")
    List<MovieModel> listMovies;

    @SerializedName("total_results")
    int totalResults;

    public List<MovieModel> getListMovies() {
        return listMovies;
    }

    public int getTotalResults() {
        return totalResults;
    }
}

File: app/src/main/java/com/example/dagger_hilt/data/datasource/web/models/MoviesResponse.kt
package com.example.dagger_hilt.data.datasource.web.models

import com.google.gson.annotations.SerializedName
import com.example.dagger_hilt.data.datasource.web.models.MovieModel

data class MoviesResponse (
    @SerializedName("results")
    val listMovies: List<MovieModel>,

    @SerializedName("total_results")
    val totalResults: Int
)